wgsl/index.bs

```diff
@@ -919,16 +920,27 @@ Note: A type assertion is a statement of fact about the text of a program.
 It is not a runtime check.
 
 Finding static types for expressions can be performed by recursively applying type rules.
-A <dfn noexport>type rule</dfn> has two parts:
-* A conclusion, stated as a type assertion for an expression.
-    The expression in the type assertion is specified schematically,
-    using *italicized* names to denote subexpressions
-    or other syntactically-determined parameters.
-* Preconditions, consisting of:
+A <dfn noexport>type rule</dfn> has three parts:
+* A <dfn noexport lt="type rule conclusion">Conclusion</dfn>, stated as a [=type assertion=] for an expression.
+    * The expression in the type assertion is specified schematically,
+        using *italicized* names to denote subexpressions
+        or other syntactically-determined parameters.
+* <dfn noexport lt="type rule preconditions">Preconditions</dfn>, consisting of:
     * Type assertions for subexpressions, when there are subexpressions.
     * Conditions on the other schematic parameters, if any.
     * How the expression is used in a statement.
     * Optionally, other static context.
+* <dfn noexport lt="semantics of an expression">Semantics</dfn>, which is the effect of evaluating the expression,
```

Comment:
There are many inconsistencies in the various type tables. Some call these notes (which I think is particularly problematic for this change), description or semantics. I would suggest making the tables consistent (use either description or semantics) and updating this name to match (if description is used).